src/Components/Navbar/nav.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+const NavBar = () => {
```

Comment:
Consider separating it with an empty new line.

Reply:
I've added new line between import and function
![image](https://user-images.githubusercontent.com/46630057/74604837-c2d03280-50d2-11ea-8a7e-7431d7d9c5a0.png)